bin/git-recent

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+
+usage() {
+    echo 
+    echo "Usage: git recent"
+    echo "Prints a nice summary of the latest $COMMIT_SUMMARY_LENGTH commits:"
+    echo 
+    ## Not sure what format of log to use...
+    #git log -"$COMMIT_SUMMARY_LENGTH" --pretty=format:"%h - %an, %ar : %s"
+    echo 
+    exit $1
+}
+
+if [[ $# -ne 1 ]]; then
+        COMMIT_SUMMARY_LENGTH=12
+    else
+        COMMIT_SUMMARY_LENGTH=$1
+fi
+
+git log -"$COMMIT_SUMMARY_LENGTH" --graph --pretty=format:'%Cred%h%Creset %s%C(yellow)%d%Creset %Cgreen(%cr)%Creset [%an]' --abbrev-commit --date=relative
```

Comment:
This is a completely new operation that some of the `usage()` functions will rely on to help users by pretty printing recent commits.